Gems/PhysX/Code/Include/PhysX/Joint/PhysXJointBus.h

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+
+#pragma once
+
+#include <AzCore/Component/EntityId.h>
+#include <AzCore/EBus/EBus.h>
+#include <AzCore/Interface/Interface.h>
+#include <AzCore/std/utils.h>
+#include <AzCore/Math/Transform.h>
+
+namespace PhysX
+{
+    //! Interface to communicate with PhysX joint's motor.
+    class JointInterface : public AZ::EBusTraits
+    {
+    public:
+        static constexpr AZ::EBusAddressPolicy AddressPolicy = AZ::EBusAddressPolicy::ById;
+        using BusIdType = AZ::EntityId;
+
+        virtual ~JointInterface() = default;
+
+        //! Returns current position from joint
+        virtual float GetPosition() = 0;
```

Comment:
How come `GetPosition` only returns a single float? Shouldn't return a AZ::Vector3? What kind of position is this?

Reply:
````
        //! It is a relative position of the entity in the direction of the free axis of the joint.
        //! For a hinge joint it is a twist angle (in radians), for the prismatic joint it is travel (in meters).
````

Awesome! thanks for the comment, it's very clear now.